catboost/python-package/catboost/core.py

```diff
@@ -5826,3 +5826,71 @@ def to_classifier(model):
     for attr in model.__dict__:
         setattr(classifier, attr, getattr(model, attr))
     return classifier
+
+
+from catboost.utils import read_cd
+
+
```

Comment:
catboost.Pool should support objects of ColumnDescription passed in column_description argument of its constructor